net.sf.eclipsecs.core/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-ClassPath: .,
  lib/dom4j-2.1.3.jar,
  lib/snakeyaml-1.26.jar,
  lib/reflections-0.9.10.jar,
- lib/metadata-gen-1.0-SNAPSHOT.jar
+ lib/checkstyle-8.36-all.jar
```

Comment:
I am not sure why "-all" jar become required here ... It should work as it was before, we reuses the same "all.jar"
Remove lib/reflections-0.9.10.jar also.

Reply:
Its because the JAR is present in checkstyle, but we also need it in `core` package, since we call the library methods in `MetadataFactory`.